functional_testing/Grid_API_Testing/api_testing/testcases/basic_tests/test_storagepools_apis.py

```diff
@@ -1,8 +1,8 @@
-from random import randint
+import  random
 from api_testing.testcases.testcases_base import TestcasesBase
 from api_testing.grid_apis.apis.storagepools_apis import StoragepoolsAPI
-
-
+from api_testing.python_client.client import Client
+import unittest, time
 
 class TestStoragepoolsAPI(TestcasesBase):
```

Comment:
please add verifying methods using the client.btrfs as every storagepool API is implemented using this it.
to knw which btrfs argument u need to knw please check the following link:
https://github.com/Jumpscale/jumpscale_core8/blob/8.2.0/lib/JumpScale/sal/g8os/StoragePool.py